src/lib/Select/Select.svelte

```diff
@@ -285,6 +285,10 @@
     word-wrap: var(--non-selected-word-break, break-word);
     white-space: var(--non-selected-white-space);
     font-weight: var(--non-select-font-weight, 500);
+    left: var(--select-left);
```

Comment:
name these props with consistency as the class they are placed in.

Reply:
Resolved